client/src/Components/Header.jsx

```diff
@@ -7,6 +7,16 @@ const Header = () => {
 	const navigate = useNavigate();
 	const location = useLocation();
 	const { handleSignUp, handleSignOut, isLoggedIn } = useAuth();
+	const [value, setValue] = React.useState(0);
+
+	const handleChange = (e, value)=> {
+		if (value === 0) {
+			navigate("/");
+		} else if (value ===2) {
+			navigate ("/subscription");
+		}
+		setValue(value);
+	};
```

Comment:
this is an anti pattern called `magic numbers`. At the very least move the numbers to consts and give them a descriptive name. But really we want a descriptive string.

```javascript
        const homePageCode = 0
        const subscriptionPageCode = 2
	const handleChange = (e, value)=> {
		if (value === homePageCode) {
			navigate("/");
		} else if (value === subscriptionPageCode) {
			navigate ("/subscription");
		}
		setValue(value);
	};
```

Reply:
https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544

Reply:
done